ext_emconf.php

```diff
@@ -8,12 +8,12 @@
     'uploadfolder' => 0,
     'createDirs' => '',
     'clearCacheOnLoad' => 0,
-    'version' => '8.0.0',
```

Comment:
please do not change the version number in casual fixes. will do that in dedicated release commits. Thanks.